src/Datasource/Paging/NumericPaginator.php

```diff
@@ -255,6 +257,20 @@ protected function extractData(RepositoryInterface $object, array $params, array
     {
         $alias = $object->getAlias();
         $defaults = $this->getDefaults($alias, $settings);
+
+        $validSettings = array_merge(
+            array_keys($this->_defaultConfig),
+            ['whitelist', 'sortWhitelist', 'order']
```

Comment:
I had to keep `order` as a special case since we use it's value to set the default sort/direction when generating sorting links.